test/unit/annotation_spec.js

```diff
@@ -343,6 +411,154 @@ describe('annotation', function() {
 
       expect(markupAnnotation.creationDate).toEqual(null);
     });
+
+    it('should correctly parse T and not ' +
```

Comment:
Let's make this test only about not parsing IRT/RT, so change this to `should not parse IRT/RT when not defined` and remove the `T` test since that is not really related to IRT/RT not being parsed.